src/utils/constant/images.js

```diff
@@ -3,15 +3,18 @@ import search from "assets/images/icons/search.png";
 import deleteIcon from "assets/images/icons/deleteIcon.png";
 import completeIcon from "assets/images/icons/completeIcon.png";
 import editIcon from "assets/images/icons/editIcon.png";
+import addIcon from "assets/images/icons/addIcon.png";
 
 export const LOGO = logo;
 export const SEARCH = search;
 export const DELETE = deleteIcon;
 export const COMPLETE = completeIcon;
 export const EDIT = editIcon;
+export const ADD = addIcon;
```

Comment:
Could you please add context as prefix ?
```suggestion
export const ICON_ADD = addIcon;
```